kong.conf.default

```diff
@@ -48,6 +48,13 @@
 # NGINX
 #------------------------------------------------------------------------------
 
+#set_real_ip_from = 0.0.0.0/0;   # Defines trusted addresses that are known to send
+				 # correct replacement addresses. 
+				 # If the special value unix: is specified, 
+				 # all UNIX-domain sockets will be trusted.
+				 # See: http://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from
```

Comment:
Yes that's right, I missed the `;`. This will certainly cause an error because when it's being added with `set_real_ip_from ${{SET_REAL_IP_FROM}};` it basically appends a double semicolon.